assets/templates/titleAssertions.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * If you wanted to learn more about describe interface. Refer this :
+ * https://nightwatchjs.org/guide/writing-tests/test-syntax.html
+ */
+describe('Title Assertion', () => {
+    
+    /**
+     * This section will always execute before the test suite
+     * Read More : https://nightwatchjs.org/guide/writing-tests/using-test-hooks.html#before-beforeeach-after-aftereach
+     */ 
+    before(browser => {
+         /**
+         * Navigate to a URL :
+         *   - We need to navigate before performing any actions on the page
+         *     Read More : https://nightwatchjs.org/api/navigateTo.html
+         * 
+         *   - <PAGE-URL> is a placeholder. Replace it with the actual URL, which you wanted to navigate
```

Comment:
this also should have example

Reply:
I agree. If the user does not have a website they can fill the template test for, we should provide one in the example.

Also, filling in all the values from the example should complete the template test. The template tests doesn't make sense if the user cannot run it.

Reply:
Added #[f88a21b](https://github.com/nightwatchjs/create-nightwatch/pull/25/commits/f88a21b2175a4bb99bee4932926c120db3c595eb)

Reply:
Ohh, I was actually talking about the `login.js` file.

Reply:
#[0e52e21](https://github.com/nightwatchjs/create-nightwatch/pull/25/commits/0e52e21d9f55c2fea8f030173a15913066ac12b0)